iceoryx_hoofs/testing/include/iceoryx_hoofs/testing/testing_logger.hpp

```diff
@@ -34,12 +34,12 @@ namespace testing
 /// also be used to check for the occurrence on specific log messages, e.g. when a function is expected to log an error.
 /// @code
 /// callToFunctionWhichLogsAnError();
-/// if (iox::testing::TestingLogger::doesLoggerSupportLogLevel(iox::log::LogLevel::ERROR))
-/// {
-///     auto logMessages = iox::testing::TestingLogger::getLogMessages();
+/// iox::testing::TestingLogger::checkLogMessageIfLogLevelIsSupported(iox::log::LogLevel::ERROR, [](const auto&
+/// logMessages){
+///     ASSERT_GT(logMessages.size(), 0);
 ///     ASSERT_THAT(logMessages.size(), Eq(1U));
 ///     EXPECT_THAT(logMessages[0], HasSubstr(expectedOutput));
-/// }
+/// });
```

Comment:
I think the `ASSERT_GT` is superfluous

Reply:
@elBoberido Woops. Fixed ! Need to work out that link issue though...

Reply:
The links are working but it seems they have DDOS detection and refuse to respond to requests from github. Let's see if this continues next year. If the check for autosar links keeps failing we need to disable that check.